File: benchmark/results/ts-multilang/run-003/a-mcp/output/src/domain/analytics/services/metrics-aggregator.ts
import { Event } from '../aggregates/event';

/** MetricsAggregator service. */
export class MetricsAggregator {

  create(input: Partial<Event>): Event { throw new Error('not implemented'); }

  findById(id: string): Event | null { throw new Error('not implemented'); }

  list(limit: number): Event[] { throw new Error('not implemented'); }

  delete(id: string): void { throw new Error('not implemented'); }
}

File: benchmark/results/java-invocation-script/run-004/b-baseline/output/com/metaengine/demo/analytics/services/MetricsAggregator.java
package com.metaengine.demo.analytics.services;

import com.metaengine.demo.analytics.aggregates.Event;
import java.util.List;

/** MetricsAggregator domain service in the analytics domain. */
public class MetricsAggregator {
    /** Aggregate metrics from an Event partial input. */
    public Event create(Event input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Event by id, or null if not found. */
    public Event findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Events up to limit. */
    public List<Event> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Event by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
